src/tools/exports/structuresNombreDePosteEnTrop.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+'use strict';
+const path = require('path');
+const fs = require('fs');
+
+const { execute } = require('../utils');
+const utils = require('../../utils/index');
+
+execute(__filename, async ({ logger, db }) => {
+  const users = await db.collection('users').find({ roles: { $elemMatch: { $eq: 'structure' } } }).limit(5).toArray();
```

Comment:
ok ou + simple` roles: { $in: ['structure'] }`

Reply:
Pk le limit 5 ici ?

Reply:
limit(5) ?

Reply:
ha la limit, c'est un oublie, pour tester  avec quelque un  en local...